news/feature-3754.md

```diff
@@ -0,0 +1,10 @@
+`disk-buffer`: Added a new option to reliable disk-buffer: `qout-size()`.
```

Comment:
I've also reviewed this PR and I'm ready to approve it.
Since @alltilla is on vacation, I'm adding the default value (which will be 1000 after merging #3757) to the news entry.
To keep news entries short, I would not mention more.

The minimum value is 0, which is forced by our grammar.
There is no theoretical maximum value, but setting this larger than the disk buffer's size would not make sense (but it is allowed and it will work correctly).

Reply:
There is an extra check in the setter, that enforces the 64 as min value. Maybe allowing 0 wouldn't be a bad thing. What do you think ?

Reply:
We allow 0 with "nonnegative_integer".

Reply:
Oh sorry, I misunderstood the comment about 64. Yeah, we should definitely allow 0.

Reply:
Done.